### DISCUSSION TITLE
[Projects Beta] - Listing Tasks in Issue

### DISCUSSION THREAD

**Discussion by theilgaz** [community/community#7575] [Projects and Issues] [answered]
- [x] Title field can be used for displaying current tasks status or maybe a new field to show them. _(1 of 4 tasks)_

![issues](https://user-images.githubusercontent.com/13048645/141249839-0042fc2b-5090-4ac3-8b48-028ac3bb7878.png)

**Comment by eleijonmarck** [comment]
![9EF1A7B3-90FD-463D-B7A1-C955B66AB96A_4_5005_c](https://user-images.githubusercontent.com/7727602/143243140-2423e361-66a2-4c62-8bd3-a4c2753d2f0a.jpeg)
would be great to get the task list in the board view as well


the issue you see here has 2 tasks in the issue

**Reply by theilgaz** [reply]
Definitely! Tasks in Issue is a true lifesaver for tracking stories.

**Comment by timlevett** [comment]
+1 on the board view. Also for the table views, I would like this as an independent field like assignee.

![image](https://user-images.githubusercontent.com/3534544/143257067-e87fb068-cd41-4120-ba5c-4bcae35a1718.png)

**Reply by andre-wells** [reply]
What would having the task list as an independent field serve?  It's not an item you could group by, and sorting to lest complete to most complete doesn't give enough context (in my opinion).

**Reply by theilgaz** [reply]
I like the idea of another field. There is a need for a task list in Projects.

**Reply by jensf** [reply]
I would very much like this feature in projects - and I'd rather have in the title than not at all. 

@andre-wells - good question
>Why as a independent field?

- Perhaps for some view scenarios the # tasks is NOT important, so this allows it to be removed.
- Sorting to most complete is sometimes helpful to find items that are almost done to finish up.

**Reply by omerbensaadon** [reply]
This is exactly what we've done with the new Tasklists! 

Sign up for the feature preview at https://github.com/features/preview

**Comment by fydar** [comment]
Give me tasks in the table view, pweese! I don't care about sorting or grouping, I just want **pretty tables** :3 

Below is a proposal for the sort of thing I would love to see.

![image](https://user-images.githubusercontent.com/19309165/147187531-e7ff3179-d1f0-40d7-a633-bc65d9a220e6.png)

**Reply by fydar** [reply]
Potentially grouping could be done on a **no-tasks**, **in-progress tasks**, and **all tasks complete** basis; or make a column that can't be used in the grouping. I'm sure it won't be the last.

**Reply by omerbensaadon** [reply]
so this exists with our new version of Tasklists! You can join the preview here: https://github.com/features/preview

**Comment by timharris777** [comment]
Agreed. I'm missing task progress on table or board view of issues.

**Comment by Neudrino** [comment]
[This present issue](https://github.com/github-community/community/discussions/7575) is probably most popular issue with regard to tasks in Github Projects (Beta)!

Other (possible) duplicates are:
- https://github.com/github-community/community/discussions/16616
- https://github.com/github-community/community/discussions/13033
- https://github.com/github-community/community/discussions/16721
- https://github.com/github-community/community/discussions/12050
- https://github.com/github-community/community/discussions/15929

An **answer** from Github as of 2022-06-13 can be found at https://github.com/github-community/community/discussions/16721#discussioncomment-2939031.

**Reply by theilgaz** [reply]
It seems like my post was from 2021. Your possible duplicates were from 2022.

**Comment by omerbensaadon** [comment, marked as answer]
Hey! We're working on this. See this comment for the latest: https://github.com/orgs/community/discussions/16721#discussioncomment-2939031

Thank you @Neudrino for your help finding dupes!

**Reply by theilgaz** [reply]
Great news! Thank you.

By the way, my post was **the first one** to point out this issue, fyi. I've never submit before checking duplicate, anytime, any platform.

**Comment by oscarmanzelli** [comment]
Any news on that?

**Comment by Neudrino** [comment]
For anyone wondering what happened to "end of year 2022":
See https://github.com/github/roadmap/issues/760.
![grafik](https://github.com/community/community/assets/13189449/e6655197-7374-4015-8c7a-5cba7f9d5ba0)

**Comment by taoliujun** [comment]
I remember back in the day, the issue list has task progress in front of title, but now it is just a task icon.

![image](https://github.com/user-attachments/assets/1ba587fc-a299-482b-81c5-9e7a468e2db2)
